static/app/data/platformCategories.tsx

```diff
@@ -338,3 +338,6 @@ export const replayOnboardingPlatforms: readonly PlatformKey[] = [
   'javascript-vue',
   'javascript',
 ];
+
+// These are the supported platforms that can also be set up using the loader
+export const replayPlatformsLoader: readonly PlatformKey[] = ['javascript'];
```

Comment:
I find this name is a bit confusing: It loads replay platforms?

How about `replayLoaderInstallationPlatformList` or something like that?